docs/README.md

```diff
@@ -1,28 +1,51 @@
-# Skript Documentation Templates
+# Skript Documentation
 
-Skript's features are documented directly in it's Java code. But we still need
+Here is the source code of [Skript website](https://skriptlang.github.io/Skript/).
 
-1. HTML, CSS and (possible) Javascript code to create website out of these
-2. Clear tutorials, not just "you can check the syntax pattern"
-3. Examples explained, if needed
+What the website uses:
 
-When generating final result, each HTML file is surrounded by template.html,
-which provides head element, navigation bar and so on.
+ * Bulma
+ * FontAwesome
+ * Svelte
+ * Sapper
 
-## Template Patterns
+## Export Documentation
 
-Patterns have syntax of ${pattern_here}. For example, ${skript.version} is replaced with
-current Skript version. Please see below for more...
+To export the documentation, run these commands in your terminal:
 
-You can also include other files by using ${include <filename>}. Just please make
-sure that those included files don't have tags which are not allowed in position
-where include is called.
+**Using NPM:**
```

Comment:
People exporting documentation should not be required to install anything in addition to Minecraft server + Java.

Reply:
I'll clarify this a bit: currently, on Linux, Windows and Mac, Skript needs JDK 8 or newer to be compiled. Gradle wrapper provides Gradle, which in turn provides all compile-time dependencies. So, only JDK needed to produce a usable jar.

Spigot installation is, at the moment, needed for generating documentation. We could probably get rid of this, though, and just prepare classpath with dependencies (Guava, etc.) in build script.

Reply:
Documentation should be not related to any minecraft server

Reply:
the doc site is generated within the plugin itself, so it runs where the plugin runs (a minecraft server)